src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/DecisionAcceptedEmailService.java

```diff
@@ -2,6 +2,7 @@
 
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseDetails;
+import uk.gov.hmcts.reform.sscscorbackend.domain.TribunalViewResponse;
 
 @Service
 public class DecisionAcceptedEmailService {
```

Comment:
Should this now be decision received service

Reply:
Yes thought that as I was doing this and then forgot to change it will rename.